src/main/java/com/bynder/sdk/sample/AppSample.java

```diff
@@ -13,77 +13,121 @@
 import com.bynder.sdk.model.Media;
 import com.bynder.sdk.model.MediaType;
 import com.bynder.sdk.model.oauth.RefreshTokenCallback;
-import com.bynder.sdk.model.oauth.Token;
 import com.bynder.sdk.query.MediaQuery;
 import com.bynder.sdk.query.OrderBy;
 import com.bynder.sdk.service.BynderClient;
 import com.bynder.sdk.service.asset.AssetService;
 import com.bynder.sdk.service.oauth.OAuthService;
+import com.bynder.sdk.util.Utils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.awt.*;
 import java.io.IOException;
+import java.net.MalformedURLException;
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.net.URL;
 import java.util.Arrays;
 import java.util.List;
+import java.util.Properties;
 import java.util.Scanner;
 
 /**
  * Sample class to display some of the SDK functionality.
  */
 public class AppSample {
```

Comment:
I'm not sure why this file is involved in this PR

Reply:
Could have only done the change of using the `Utils.loadConfig()` method. But I had this code already anyway. Could move it to a separate PR but personally didn't feel like this PR wasn't too complex anyway.

Reply:
Moved it out of this PR